tests/cases/fourslash/renameNumericalIndex.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference path="fourslash.ts" />
+
+////const foo = { 0: true };

```

Comment:
```suggestion
////const foo = { [|0|]: true };
```

Reply:
@sheetalkamat Could you please explain this change?

Reply:
test rename when invoked at this location too